h2o-py/h2o/tree/tree.py

```diff
@@ -5,24 +5,63 @@
+    
+    Every graph starts wit ha root node, which is in fact an instance of H2OSplitNode and is naturally the very beginning of all decision paths in graph.
+    It points to its children, which again point to their children, unless an H2OLeafNode is hit. Nodes are always numbered from the top of the tree down to the lowest level, from left to right.
+    
+    At tree level, the following information is provided:
+    - Number of nodes in the tree,
+    - Model the tree belongs to,
+    - Tree class (if applicable),
+    - Pointer to a root node for tree traversal (breadth-first, depth-first) and manual tree walking.
+    
+    Each node in the tree is uniquely identified by an ID, regardless of its type. Also for each node type, a human-redable description is available. There are two types nodes distinguished:
+    - Split node,
+    - Leaf node.
+    
+    -------------    
+    A split node is a single non-terminal node with either numerical or categorical feature split. The root node is guaranteed to be a split node, as a zero-depth tree t of cardinality |t| = 1 contains no decisions at all. 
+    Every split node consists of:
+    
+    1. H2O-specific node identifier (ID - all nodes have it),
+    2. Left child node & right child node,
+    3. Split feature name (split column name),
+    4. Split threshold (mainly for numerical splits),
+    5. Categorical features split (categorical splits only),
+    6. Direction of NA values (which way NA values go - left child, right child or nowhere).
+    
+    -------------
+    A leaf node is a single node with no children, thus being a terminal node at the end of the decision path in a tree. Leaf node consists of:
+    1. H2O-specific node identifier (ID - all nodes have it),
+    2. Prediction value (floating point number).
+    
     :param model: The model this tree is related to.
     :param tree_number: An integer representing the order in which the tree has been built in the model.
-    :param tree_class: A string representing the name of the tree's class. The number of tree classes equals the number of levels in categorical response column. As there is exactly one class per categorical level, the name of the tree's class is equal to the corresponding categorical level of the response column. In case of regression and binomial models, the name of the categorical level is ignored and can be omitted, as there is exactly one tree built in both cases.
+    :param tree_class: A string representing the name of the tree's class. Specifies the class of the tree requested. Required for multi-class classification. The number of tree classes equals the number of levels in categorical response column. As there is exactly one class per categorical level, the name of the tree's class is equal to the corresponding categorical level of the response column. In case of regression and binomial models, the name of the categorical level is ignored and can be omitted.
```

Comment:
I am thinking:
:param tree_class: A string representing the name of the **target** class. Specifies the......

Reply:
The reason I would like to avoid the words "tree's class" is that this object is called "H2O Tree Class" so it can be confusing to the reader using the same name.